tests/test_vpc.rb

```diff
@@ -87,7 +88,7 @@ def test_auto_recovery_reload_delay
     assert_equal(300, @vpc.auto_recovery_reload_delay,
                  'Auto recovery delay should be 300')
     # negative high range
-    e = assert_raises(CliError) { @vpc.auto_recovery_reload_delay = 3601 }
+    e = assert_raises(CliError) { @vpc.auto_recovery_reload_delay = 86_401 }
```

Comment:
Why test ranges at all?  Don't see much value since the device will reject invalid ranges anyway and this could differ across platforms.  In general our strategy has been to allow the device to reject things like this instead of explicitly checking for them in puppet.  I suggest you just remove this test unless there is another good reason for keeping it.